feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -65,16 +71,69 @@ class AmazonBilling constructor(
     override fun endConnection() { }
 
     override fun queryAllPurchases(
+        appUserID: String,
         onReceivePurchaseHistory: (List<PurchaseHistoryRecordWrapper>) -> Unit,
         onReceivePurchaseHistoryError: PurchasesErrorCallback
     ) {
-        // TODO
+        purchaseUpdatesHandler.queryPurchases(
+            onSuccess = { receipts, userData ->
+                if (receipts.isEmpty()) {
+                    onReceivePurchaseHistory(emptyList())
+                    return@queryPurchases
+                }
+                getMissingSkusForReceipts(
+                    appUserID,
+                    userData.userId,
```

Comment:
I've been using them more and more. I try to do this https://kotlinlang.org/docs/reference/coding-conventions.html#named-arguments 

> Use the named argument syntax when a method takes multiple parameters of the same primitive type, or for parameters of Boolean type, unless the meaning of all parameters is absolutely clear from context.

Which is kinda subjective. I will add here the `amazonUserID` name argument since `receipts` is clear.